tool/generate_symbols_table.dart

```diff
@@ -103,6 +103,13 @@ Future<void> main() async {
     '  }',
     '}',
     '',
+    'extension SymName2 on String {',
```

Comment:
Might it not be better to do a static method `SymFromString(String) -> Sym`?

I wish we could static extensions like `Sym.fromString`, hehe..

But an extension on `String` seems a bit dangerous.